### PR TITLE
Change README to favor running in Docker

### PR DESCRIPTION
**Pull request by kjetiljd**
The tooling is old and it is just easier to run in Docker at the moment.

Merge PR 58 and 59 before this. 

This arguably resolves issue #46 .

### CODE REVIEW

**Comment by mattwynne** [comment]
@rchatley I propose we give @kjetiljd commit rights, he's doing good stuff! Do you agree?

**Comment by rchatley** [comment]
Hi both (@mattwynne , @kjetiljd ) - sorry for the slow response the last week or so has been super busy for me, so I haven't been paying much attention.

Yes, that sounds good to me. I haven't really spent much effort on updating Extreme Startup to keep pace with tooling updates etc, but I've still be using it to run some workshops, so having a bit more momentum on the development would be great.

I'll add @kjetiljd now.

**Comment by kjetiljd** [comment]
@rchatley - sorry, seems I found the invite to committer after it expired. If you issue another I promise to get it right this time...

**Comment by rchatley** [comment]
No worries, let me try again now.
